src/pretix/base/models/event.py

```diff
@@ -544,6 +546,59 @@ def get_users_with_permission(self, permission):
             Q(is_superuser=True) | Q(twp=True)
         )
 
+    @staticmethod
+    def has_paid_things(event):
```

Comment:
Is there any reason this method is static? ``event.has_paid_things()`` looks cleaner to me than ``Event.has_paid_things(event)``.